run_benchmarks.rb

```diff
@@ -11,6 +11,7 @@
 require 'etc'
 require 'yaml'
 require_relative 'misc/stats'
+require_relative 'misc/benchmark_mode'
```

Comment:
Unlike `misc/graph.rb` that supports both standalone execution and use as a library or other standalone scripts, these two files (`misc/stats.rb` and `misc/benchmark_mode.rb`) only make sense as a library for `run_benchmarks.rb`. So it feels confusing to me to have them under `misc/`. WDYT about moving `misc/stats.rb` and `misc/benchmark_mode.rb` to `lib/`?